torch/_dynamo/variables/user_defined.py

```diff
@@ -523,7 +523,7 @@ def is_standard_new(self):
         new_fn = inspect.getattr_static(self.value, "__new__", None)
         if isinstance(new_fn, staticmethod):
             new_fn = new_fn.__func__
-        return new_fn in (object.__new__, Generic.__new__)
+        return new_fn in {object.__new__, Generic.__new__}
```

Comment:
Why this change?  Using a set like this will only be faster if you move it into global scope, since you still need to construct the set on every call to the function (which is O(n), but also involves a bunch of hashing so is higher slower than tuple creation).

Reply:
> Using a set like this will only be faster if you move it into global scope

That makes sense. Reverted.